RetrospectiveExtension.Frontend/components/feedbackBoardContainer.tsx

```diff
@@ -181,6 +182,9 @@ export default class FeedbackBoardContainer extends React.Component<FeedbackBoar
         setTimeout(this.tryReconnectToBackend, 2000);
       });
 
+      const isHosted = await isHostedAzureDevOps();
+      this.setState({ isHostedAzureDevOps: isHosted });
```

Comment:
instead of setting default value for the `isHostedAzureDevOps` state and calculate it and setting the state afterwards, can we do the calculation beforehand and use it when we set the default state values?

something like (in pseudo);

- calculate if it's hosted on azure devops
- set the default state

it could save a re-render due to the extra state update

Reply:
I'll move it to index.tsx file.

Reply:
Fixing it by moving `isHostedAzureDevOps` to props from `index.tsx` file.